docs/serverless/README.md

```diff
@@ -0,0 +1,53 @@
+# Serverless Agent
+
+The Serverless Agent is a build of the Datadog Agent capable of running
+in serverless environments, to collect custom metrics, the logs generated by the
+function and the traces of the function execution. Today, its main usage is to
+build the Datadog Lambda Extension for AWS Lambda.
+
+## Concepts
+
+Today, the Serverless Agent is intended to run as a side-car of the serverless function.
+
+### Validate the presence of the Serverless Agent and order a flush
+
+The Serverless Agent runs an HTTP server on port 8124.
+
+With the Datadog Lambda Extension, this HTTP server listens for two
+different routes:
+    - /lambda/hello
+    - /lambda/flush
+
+These routes are not doing anything in specialized for AWS, however, they can be
+at some point. That's the reason that if the Serverless Agent has to support a new
+serverless environment, it would be important to create different HTTP routes (most
```

Comment:
```suggestion
serverless environment, like GCP or Azure App Services Extension, it would be important to create different HTTP routes (most
```
is this what you mean here? if not, please update with examples of what you do mean

Reply:
I'll use

`If the Serverless Agent needs to be set up to support a different serverless environment, on GCP or Azure for instance, it would be`

if that sounds good to you.

Reply:
yep, perfect. I just wasn't clear here if you meant a completely different service here or a multiple of the same service